### PR TITLE
KOGITO-8883: Integrate and test the devui with the embedded extension…

### PR DESCRIPTION
**Pull request by nmirasch**
…s Data Index and Job Service
jira: https://issues.redhat.com/browse/KOGITO-8883

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- for <b>pull request checks</b>  
  Please add comment: <b>Jenkins retest this</b>

- for a <b>specific pull request check</b>  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] tests</b>

- for <b>quarkus branch checks</b>  
  Run checks against Quarkus current used branch  
  Please add comment: <b>Jenkins run quarkus-branch</b>

- for a <b>quarkus branch specific check</b>  
  Run checks against Quarkus current used branch  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] quarkus-branch</b>

- for <b>quarkus main checks</b>  
  Run checks against Quarkus main branch  
  Please add comment: <b>Jenkins run quarkus-main</b>

- for a <b>specific quarkus main check</b>  
  Run checks against Quarkus main branch  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] quarkus-main</b>

- for <b>quarkus lts checks</b>  
  Run checks against Quarkus lts branch  
  Please add comment: <b>Jenkins run quarkus-lts</b>

- for a <b>specific quarkus lts check</b>  
  Run checks against Quarkus lts branch  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] quarkus-lts</b>
 
- for <b>native checks</b>  
  Run native checks  
  Please add comment: <b>Jenkins run native</b>

- for a <b>specific native check</b>  
  Run native checks 
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] native</b>

- for <b>native lts checks</b>  
  Run native checks against quarkus lts branch
  Please add comment: <b>Jenkins run native-lts</b>

- for a <b>specific native lts check</b>  
  Run native checks against quarkus lts branch
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] native-lts</b>
 
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: data-index/data-index-common/src/main/resources/basic.schema.graphqls [hunk 1]
```diff
@@ -354,4 +355,65 @@ enum OrderBy {
 input Pagination {
     limit: Int
     offset: Int
+}
+
+type Job {
```
Comment (wmedvede) [inline data-index/data-index-common/src/main/resources/basic.schema.graphqls:360]: Silly question, so now we ensure the Job related queries are always defined, right?

**Review by wmedvede — APPROVED** [review]
LGTM.

**Review by cristianonicolai — APPROVED** [review]

File: data-index/data-index-common/src/main/resources/basic.schema.graphqls (hunk 1 above)
Comment (nmirasch) [reply data-index/data-index-common/src/main/resources/basic.schema.graphqls:360]: @wmedvede yes, this code is related to add the Job definition at the GraphQL schema, making them available through GraphQL queries.

File: data-index/kogito-addons-quarkus-data-index/kogito-addons-quarkus-data-index-common/runtime/src/main/java/org/kie/kogito/index/addon/DataIndexEventPublisher.java [hunk 2]
```diff
@@ -49,6 +53,13 @@ public void publish(DataEvent<?> event) {
             case "UserTaskInstanceEvent":
                 indexingService.indexUserTaskInstance(new UserTaskInstanceEventMapper().apply((UserTaskInstanceDataEvent) event));
                 break;
+            case "JobEvent":
+                try {
+                    indexingService.indexJob(getObjectMapper().readValue(new String((byte[]) event.getData()), Job.class));
+                } catch (IOException e) {
+                    LOGGER.error("Error mapping JobEvent", e);
```
Comment (radtriste) [inline data-index/kogito-addons-quarkus-data-index/kogito-addons-quarkus-data-index-common/runtime/src/main/java/org/kie/kogito/index/addon/DataIndexEventPublisher.java]: If we can't index, we just ignore ?

File: data-index/kogito-addons-quarkus-data-index/kogito-addons-quarkus-data-index-common/runtime/src/test/java/org/kie/kogito/index/addon/DataIndexEventPublisherTest.java [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        ProcessInstanceDataEvent event = getProcessCloudEvent(PROCESS_ID, PROCESS_INSTANCE_ID, COMPLETED,
+                ROOT_PROCESS_INSTANCE_ID, ROOT_PROCESS_ID, ROOT_PROCESS_INSTANCE_ID);
+
+        dataIndexEventPublisher.publish(event);
+
+        verify(indexingService).indexProcessInstance(eventCaptor.capture());
+    }
+
+    @Test
+    void onJobEvent() throws Exception {
```
Comment (radtriste) [inline data-index/kogito-addons-quarkus-data-index/kogito-addons-quarkus-data-index-common/runtime/src/test/java/org/kie/kogito/index/addon/DataIndexEventPublisherTest.java:85]: would that be possible to test with a malformed JobEvent ?

**Comment by kie-ci3** [comment]
**(tests) - kogito-apps job** `#863` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure


<details>
<summary><b>Reproducer</b></summary>


export BUILD_MVN_OPTS_CURRENT=-Dvalidate-formatting -Prun-code-coverage
build-chain build cross_pr -f 'https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml' -o 'bc' -p kiegroup/kogito-apps -u https://github.com/kiegroup/kogito-apps/pull/1699 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-apps.tests.kogito-apps/863/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
Setting status of 9a115433d5fbd92fd9f464fe6a7d3ea54701c406 to PENDING with url https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-apps.tests.kogito-apps/863/display/redirect and message: 'Build started.'<br/>Using context: (build) kogito-apps<br/>Checking out git https://github.com/kiegroup/kogito-pipelines.git into /var/jenkins_home/jobs/KIE/jobs/kogito/jobs/main/jobs/pullrequest/jobs/kogito-apps.tests.kogito-apps/workspace@script/ee10b829cbd0ac763d0be6f0356db3418b2d4469e9daa1842e842313521cde92 to read .ci/jenkins/Jenkinsfile.buildchain<br/>The recommended git tool is: NONE<br/>using credential kie-ci3<br/> > git rev-parse --resolve-git-dir /var/jenkins_home/jobs/KIE/jobs/kogito/jobs/main/jobs/pullrequest/jobs/kogito-apps.tests.kogito-apps/workspace@script/ee10b829cbd0ac763d0be6f0356db3418b2d4469e9daa1842e842313521cde92/.git # timeout=10<br/>Fetching changes from the remote Git repository<br/> > git config remote.origin.url https://github.com/kiegroup/kogito-pipelines.git # timeout=10<br/>Cleaning workspace<br/> > git rev-parse --verify HEAD # timeout=10<br/>Resetting working tree<br/> > git reset --hard # timeout=10<br/> > git clean -fdx # timeout=10<br/>Fetching upstream changes from https://github.com/kiegroup/kogito-pipelines.git<br/> > git --version # timeout=10<br/> > git --version # 'git version 2.31.1'<br/>using GIT_ASKPASS to set credentials PSSWD: kie-ci3 account to access GitHub via token<br/> > git fetch --tags --force --progress -- https://github.com/kiegroup/kogito-pipelines.git +refs/heads/*:refs/remotes/origin/* # timeout=10<br/> > git rev-parse origin/main^{commit} # timeout=10<br/>Checking out Revision e35a887ac2f30b05fffb4a1da6ce9d0fbf9809d8 (origin/main)<br/> > git config core.sparsecheckout # timeout=10<br/> > git checkout -f e35a887ac2f30b05fffb4a1da6ce9d0fbf9809d8 # timeout=10<br/>Commit message: "Quarkus3: Enable PR checks (#944)"<br/> > git rev-list --no-walk 55bab850ddc9415fc4a3c2f304846aa7f37688b9 # timeout=10<br/>Loading library jenkins-pipeline-shared-libraries@main<br/>Attempting to resolve main from remote references...<br/> > git --version # timeout=10<br/> > git --version # 'git version 2.31.1'<br/>using GIT_SSH to set credentials SSH: kie-ci service account with access to jboss-integration private github space<br/>[INFO] Currently running in a labeled security context<br/> > /usr/bin/chcon --type=ssh_home_t /tmp/jenkins-gitclient-ssh5663373044418008585.key<br/>Verifying host key using manually-configured host key entries<br/> > git ls-remote -h -- https://github.com/kiegroup/jenkins-pipeline-shared-libraries # timeout=10<br/>Found match: refs/heads/main revision 928fa1ccfce5990c91a751f074d5207b7a0505eb<br/>Selected Git installation does not exist. Using Default<br/>The recommended git tool is: NONE<br/>using credential kie-ci-user-key<br/> > git rev-parse --resolve-git-dir /var/jenkins_home/jobs/KIE/jobs/kogito/jobs/main/jobs/pullrequest/jobs/kogito-apps.tests.kogito-apps/workspace@libs/d03f8b53ef89e88ec2b15428a250a4b1fb7910bfcdd3e1c33e4e95c3adb758bb/.git # timeout=10<br/>Fetching changes from the remote Git repository<br/> > git config remote.origin.url https://github.com/kiegroup/jenkins-pipeline-shared-libraries # timeout=10<br/>Fetching without tags<br/>Fetching upstream changes from https://github.com/kiegroup/jenkins-pipeline-shared-libraries<br/> > git --version # timeout=10<br/> > git --version # 'git version 2.31.1'<br/>using GIT_SSH to set credentials SSH: kie-ci service account with access to jboss-integration private github space<br/>[INFO] Currently running in a labeled security context<br/> > /usr/bin/chcon --type=ssh_home_t /var/jenkins_home/jobs/KIE/jobs/kogito/jobs/main/jobs/pullrequest/jobs/kogito-apps.tests.kogito-apps/workspace@libs/d03f8b53ef89e88ec2b15428a250a4b1fb7910bfcdd3e1c33e4e95c3adb758bb@tmp/jenkins-gitclient-ssh8936048542362754725.key<br/>Verifying host key using manually-configured host key entries<br/> > git fetch --no-tags --force --progress -- https://github.com/kiegroup/jenkins-pipeline-shared-libraries +refs/heads/*:refs/remotes/origin/* # timeout=10<br/>Checking out Revision 928fa1ccfce5990c91a751f074d5207b7a0505eb (main)<br/> > git config core.sparsecheckout # timeout=10<br/> > git checkout -f 928fa1ccfce5990c91a751f074d5207b7a0505eb # timeout=10<br/>Commit message: "[BXMSPROD-2011] Added withKerberos utility method (#270)"<br/> > git rev-list --no-walk 928fa1ccfce5990c91a751f074d5207b7a0505eb # timeout=10<br/>[Pipeline] Start of Pipeline<br/>[Pipeline] node<br/>Still waiting to schedule task<br/>Waiting for next available executor on ‘kie-rhel7&&kie-mem16g&&!built-in’<br/>Running on rhos-01-kie-psi-rhel7-xlarge-878 in /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-apps.tests.kogito-apps<br/>[Pipeline] {<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Checkout SCM)<br/>[Pipeline] checkout<br/>The recommended git tool is: NONE<br/>using credential kie-ci3<br/>Cloning the remote Git repository<br/>Cloning repository https://github.com/kiegroup/kogito-pipelines.git<br/> > git init /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-apps.tests.kogito-apps # timeout=10<br/>Fetching upstream changes from https://github.com/kiegroup/kogito-pipelines.git<br/> > git --version # timeout=10<br/> > git --version # 'git version 2.39.2'<br/>using GIT_ASKPASS to set credentials PSSWD: kie-ci3 account to access GitHub via token<br/> > git fetch --tags --force --progress -- https://github.com/kiegroup/kogito-pipelines.git +refs/heads/*:refs/remotes/origin/* # timeout=10<br/>Avoid second fetch<br/>Checking out Revision e35a887ac2f30b05fffb4a1da6ce9d0fbf9809d8 (origin/main)<br/> > git config remote.origin.url https://github.com/kiegroup/kogito-pipelines.git # timeout=10<br/> > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10<br/> > git rev-parse origin/main^{commit} # timeout=10<br/> > git config core.sparsecheckout # timeout=10<br/> > git checkout -f e35a887ac2f30b05fffb4a1da6ce9d0fbf9809d8 # timeout=10<br/>Commit message: "Quarkus3: Enable PR checks (#944)"<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] timestamps<br/>[Pipeline] {<br/>[Pipeline] timeout<br/>[2023-04-20T08:33:57.050Z] Timeout set to expire in 3 hr 0 min<br/>[Pipeline] {<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Tool Install)<br/>[Pipeline] tool<br/>[2023-04-20T08:33:57.366Z] Installing NodeJS from /var/jenkins_home/caches/nodejs/LINUX/amd64/16.2.0.tar.gz to /home/jenkins/tools/jenkins.plugins.nodejs.tools.NodeJSInstallation/nodejs-16.2.0 on rhos-01-kie-psi-rhel7-xlarge-878<br/>[Pipeline] envVarsForTool<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] stage<br/>[Pipeline] { (Initialize)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>[2023-04-20T08:34:03.464Z] + printenv<br/>[Pipeline] archiveArtifacts<br/>[2023-04-20T08:34:03.952Z] Archiving artifacts<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (check space before build)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] dir<br/>[2023-04-20T08:34:06.056Z] Running in /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-apps.tests.kogito-apps<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[2023-04-20T08:34:06.169Z] [INFO] space left on the machine<br/>[Pipeline] sh<br/>[2023-04-20T08:34:06.546Z] + df -h<br/>[2023-04-20T08:34:06.569Z] Filesystem      Size  Used Avail Use% Mounted on<br/>[2023-04-20T08:34:06.569Z] devtmpfs        7.8G     0  7.8G   0% /dev<br/>[2023-04-20T08:34:06.569Z] tmpfs           7.8G     0  7.8G   0% /dev/shm<br/>[2023-04-20T08:34:06.569Z] tmpfs           7.8G   17M  7.8G   1% /run<br/>[2023-04-20T08:34:06.569Z] tmpfs           7.8G     0  7.8G   0% /sys/fs/cgroup<br/>[2023-04-20T08:34:06.569Z] /dev/vda1        80G   20G   61G  25% /<br/>[2023-04-20T08:34:06.569Z] tmpfs           1.6G     0  1.6G   0% /run/user/1000<br/>[Pipeline] echo<br/>[2023-04-20T08:34:06.666Z] [INFO] space of /home/jenkins<br/>[Pipeline] sh<br/>[2023-04-20T08:34:07.018Z] + du -h -d1 /home/jenkins<br/>[2023-04-20T08:34:07.019Z] 4.0K	/home/jenkins/.ssh<br/>[2023-04-20T08:34:07.019Z] 0	/home/jenkins/go<br/>[2023-04-20T08:34:07.019Z] 8.0K	/home/jenkins/.vnc<br/>[2023-04-20T08:34:07.019Z] 8.0K	/home/jenkins/.m2<br/>[2023-04-20T08:34:07.019Z] 2.2G	/home/jenkins/git-repos<br/>[2023-04-20T08:34:07.019Z] 8.0K	/home/jenkins/.config<br/>[2023-04-20T08:34:07.019Z] 128M	/home/jenkins/virtenvs<br/>[2023-04-20T08:34:07.019Z] 4.0K	/home/jenkins/.cekit<br/>[2023-04-20T08:34:07.019Z] 32M	/home/jenkins/remoting<br/>[2023-04-20T08:34:07.019Z] 0	/home/jenkins/.cache<br/>[2023-04-20T08:34:07.019Z] 4.2M	/home/jenkins/workspace<br/>[2023-04-20T08:34:07.019Z] 0	/home/jenkins/.pki<br/>[2023-04-20T08:34:07.019Z] 100M	/home/jenkins/tools<br/>[2023-04-20T08:34:07.019Z] 0	/home/jenkins/caches<br/>[2023-04-20T08:34:07.019Z] 2.4G	/home/jenkins<br/>[Pipeline] echo<br/>[2023-04-20T08:34:07.066Z] [INFO] space of workspace<br/>[Pipeline] sh<br/>[2023-04-20T08:34:07.546Z] + du -h -d3 /home/jenkins/workspace<br/>[2023-04-20T08:34:07.547Z] 4.2M	/home/jenkins/workspace/KIE/kogito/main<br/>[2023-04-20T08:34:07.547Z] 4.2M	/home/jenkins/workspace/KIE/kogito<br/>[2023-04-20T08:34:07.547Z] 4.2M	/home/jenkins/workspace/KIE<br/>[2023-04-20T08:34:07.547Z] 4.2M	/home/jenkins/workspace<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Install build-chain tool)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[2023-04-20T08:34:08.951Z] [INFO] Getting build-chain version from composite action file<br/>[Pipeline] readFile<br/>[Pipeline] echo<br/>[2023-04-20T08:34:09.251Z] [INFO] build-chain version recovered '^v3'<br/>[Pipeline] sh<br/>[2023-04-20T08:34:09.646Z] + npm install -g '@kie/build-chain-action@^v3' -registry=https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/<br/>[2023-04-20T08:34:13.019Z] <br/>[2023-04-20T08:34:13.019Z] added 96 packages, and audited 97 packages in 3s<br/>[2023-04-20T08:34:13.019Z] <br/>[2023-04-20T08:34:13.019Z] 5 packages are looking for funding<br/>[2023-04-20T08:34:13.019Z]   run `npm fund` for details<br/>[2023-04-20T08:34:13.019Z] <br/>[2023-04-20T08:34:13.019Z] found 0 vulnerabilities<br/>[Pipeline] sh<br/>[2023-04-20T08:34:13.448Z] + npm list -g<br/>[2023-04-20T08:34:13.448Z] + grep build-chain<br/>[2023-04-20T08:34:14.046Z] ├── @kie/build-chain-action@3.0.25<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build projects)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[2023-04-20T08:34:15.769Z] BUILD_MVN_OPTS_CURRENT = -Dvalidate-formatting -Prun-code-coverage<br/>[Pipeline] configFileProvider<br/>[2023-04-20T08:34:15.948Z] provisioning config files...<br/>[2023-04-20T08:34:15.959Z] copy managed file [kogito_pr_settings] to file:/home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-apps.tests.kogito-apps@tmp/config14817272108085988348tmp<br/>[Pipeline] {<br/>[Pipeline] withCredentials<br/>[2023-04-20T08:34:16.144Z] Masking supported pattern matches of $GITHUB_TOKEN<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[2023-04-20T08:34:16.269Z] BUILD_MVN_OPTS =  -s /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-apps.tests.kogito-apps@tmp/config14817272108085988348tmp -Dmaven.wagon.http.ssl.insecure=true -Dmaven.test.failure.ignore=true<br/>[Pipeline] sh<br/>[2023-04-20T08:34:16.363Z] Warning: A secret was passed to "sh" using Groovy String interpolation, which is insecure.<br/>[2023-04-20T08:34:16.364Z] 		 Affected argument(s) used the following variable(s): [GITHUB_TOKEN]<br/>[2023-04-20T08:34:16.364Z] 		 See https://jenkins.io/redirect/groovy-string-interpolation for details.<br/>[2023-04-20T08:34:16.846Z] + build-chain build cross_pr --token **** -f https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml -o bc -p kiegroup/kogito-apps -u https://github.com/kiegroup/kogito-apps/pull/1699 --skipParallelCheckout<br/>[2023-04-20T08:34:17.845Z] # Executing pre section<br/>[2023-04-20T08:34:17.846Z] [command]/usr/bin/echo BUILD_MVN_OPTS= -s /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-apps.tests.kogito-apps@tmp/config14817272108085988348tmp -Dmaven.wagon.http.ssl.insecure=true -Dmaven.test.failure.ignore=true -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3<br/>[2023-04-20T08:34:17.846Z] BUILD_MVN_OPTS= -s /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-apps.tests.kogito-apps@tmp/config14817272108085988348tmp -Dmaven.wagon.http.ssl.insecure=true -Dmaven.test.failure.ignore=true -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3<br/>[2023-04-20T08:34:17.846Z] [command]/usr/bin/echo BUILD_MVN_OPTS_CURRENT=-Dvalidate-formatting -Prun-code-coverage dependency:tree<br/>[2023-04-20T08:34:17.846Z] BUILD_MVN_OPTS_CURRENT=-Dvalidate-formatting -Prun-code-coverage dependency:tree<br/>[2023-04-20T08:34:17.846Z] [command]/usr/bin/echo QUARKUS_VERSION=<br/>[2023-04-20T08:34:17.846Z] QUARKUS_VERSION=<br/>[2023-04-20T08:34:17.846Z] [command]/usr/bin/echo ENABLE_DEPLOY=<br/>[2023-04-20T08:34:17.846Z] ENABLE_DEPLOY=<br/>[2023-04-20T08:34:17.846Z]  <br/>[2023-04-20T08:34:17.846Z] # Execution Plan<br/>[2023-04-20T08:34:17.846Z] [INFO] 3 projects will be executed<br/>[2023-04-20T08:34:17.846Z] [INFO] [kiegroup/drools]<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 Level type: upstream<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 [before]<br/>[2023-04-20T08:34:17.846Z] [INFO] 		 export INTEGRATION_BRANCH=${{ env.INTEGRATION_BRANCH_UPSTREAM }}<br/>[2023-04-20T08:34:17.846Z] bash -c "if [ ! -z '${{ env.BUILD_ENVIRONMENT }}' ] && [ -f .ci/environments/update.sh ]; then .ci/environments/update.sh "${{ env.BUILD_ENVIRONMENT }}"; fi"<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 [commands]<br/>[2023-04-20T08:34:17.846Z] [INFO] 		 mvn clean install -Dquickly ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_UPSTREAM }} ${{ env.DROOLS_BUILD_MVN_OPTS_UPSTREAM }}<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 [after]<br/>[2023-04-20T08:34:17.846Z] [INFO] 		 docker system prune -f<br/>[2023-04-20T08:34:17.846Z] [INFO] [kiegroup/kogito-runtimes]<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 Level type: upstream<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 [before]<br/>[2023-04-20T08:34:17.846Z] [INFO] 		 export INTEGRATION_BRANCH=${{ env.INTEGRATION_BRANCH_UPSTREAM }}<br/>[2023-04-20T08:34:17.846Z] bash -c "if [ ! -z '${{ env.BUILD_ENVIRONMENT }}' ] && [ -f .ci/environments/update.sh ]; then .ci/environments/update.sh "${{ env.BUILD_ENVIRONMENT }}"; fi"<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 [commands]<br/>[2023-04-20T08:34:17.846Z] [INFO] 		 mvn clean install -Dquickly ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_UPSTREAM }} ${{ env.KOGITO_RUNTIMES_BUILD_MVN_OPTS_UPSTREAM }}<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 [after]<br/>[2023-04-20T08:34:17.846Z] [INFO] 		 docker system prune -f<br/>[2023-04-20T08:34:17.846Z] [INFO] [kiegroup/kogito-apps]<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 Level type: current<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 [before]<br/>[2023-04-20T08:34:17.846Z] [INFO] 		 export INTEGRATION_BRANCH=${{ env.INTEGRATION_BRANCH_CURRENT }}<br/>[2023-04-20T08:34:17.846Z] bash -c "if [ ! -z '${{ env.BUILD_ENVIRONMENT }}' ] && [ -f .ci/environments/update.sh ]; then .ci/environments/update.sh "${{ env.BUILD_ENVIRONMENT }}"; fi"<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 [commands]<br/>[2023-04-20T08:34:17.846Z] [INFO] 		 export MVN_CMD=`bash -c "if [ '${{ env.ENABLE_DEPLOY }}' = 'true' ]; then printf 'deploy ${{ env.DEPLOY_MVN_OPTS }} ${{ env.KOGITO_APPS_DEPLOY_MVN_OPTS }}'; else printf 'install'; fi"`<br/>[2023-04-20T08:34:17.846Z] mvn clean ${{ env.MVN_CMD }} ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_CURRENT }} ${{ env.KOGITO_APPS_BUILD_MVN_OPTS }}<br/>[2023-04-20T08:34:17.846Z] [INFO] 	 [after]<br/>[2023-04-20T08:34:17.846Z] [INFO] 		 docker system prune -f<br/>[2023-04-20T08:34:17.846Z]  <br/>[2023-04-20T08:34:17.846Z] # Checking out kiegroup/kogito-apps and its dependencies (3 projects in total). It can take some time.<br/>[2023-04-20T08:34:44.854Z] [INFO] kiegroup/drools checked out<br/>[2023-04-20T08:35:11.853Z] [INFO] kiegroup/kogito-runtimes checked out<br/>[2023-04-20T08:37:19.065Z] [ERROR] [kiegroup/kogito-apps] Error cloning kiegroup/kogito-apps and switching to target branch main<br/>[Pipeline] }<br/>[Pipeline] // withCredentials<br/>[Pipeline] }<br/>[2023-04-20T08:37:19.447Z] Deleting 1 temporary files<br/>[Pipeline] // configFileProvider<br/>[Pipeline] }<br/>[Pipeline] // script<br/>Post stage<br/>[Pipeline] junit<br/>[2023-04-20T08:37:19.870Z] Recording test results<br/>[2023-04-20T08:37:20.865Z] [Checks API] No suitable checks publisher found.<br/>[Pipeline] archiveArtifacts<br/>[2023-04-20T08:37:20.969Z] Archiving artifacts<br/>[2023-04-20T08:37:20.177Z] No test report files were found. Configuration error?<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>[2023-04-20T08:37:21.823Z] + rm -rf console.log<br/>[Pipeline] sh<br/>
</details>

**Comment by nmirasch** [comment]
Jenkins rerun kogito-apps tests

**Comment by nmirasch** [comment]
@radtriste could you review please? Thanks!

File: data-index/kogito-addons-quarkus-data-index/kogito-addons-quarkus-data-index-common/runtime/src/main/java/org/kie/kogito/index/addon/DataIndexEventPublisher.java (hunk 2 above)
Comment (radtriste) [reply data-index/kogito-addons-quarkus-data-index/kogito-addons-quarkus-data-index-common/runtime/src/main/java/org/kie/kogito/index/addon/DataIndexEventPublisher.java]: I think there was a misunderstanding here.
If we can't index. we still should display an error, not a debug
But shouldn't we raise an exception ?

**Comment by nmirasch** [comment]
@radtriste Added the exception propagation, in case of any error indexing the job,  could you review please? Thanks!

**Review by radtriste — APPROVED** [review]

**Pull request merged**